planetiler-core/src/main/java/com/onthegomap/planetiler/util/LayerStats.java

```diff
@@ -1,19 +1,24 @@
 package com.onthegomap.planetiler.util;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.onthegomap.planetiler.mbtiles.Mbtiles;
 import com.onthegomap.planetiler.render.RenderedFeature;
+import com.onthegomap.planetiler.writer.TileArchive;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
+import java.util.OptionalInt;
 import java.util.TreeMap;
 import java.util.concurrent.CopyOnWriteArrayList;
 import java.util.function.Consumer;
 import javax.annotation.concurrent.NotThreadSafe;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Tracks the type and zoom range of vector tile attributes that have been emitted by layer to populate the {@code json}
- * attribute in the mbtiles output metadata.
+ * Tracks the type and zoom range of vector tile attributes that have been emitted by layer to populate the
+ * {@code vector_layers} attribute in the archive output metadata. Matches the MBTiles spec, but can be reused by other
```

Comment:
For javadoc, the first line is special and gets used as a short description where the full javadoc is too much, so the convention is to do:

```java
/**
 * First sentence with short description of the method.
 * <p>
 * More details...
 */
```

For example BufferedReader.read's full javadoc generates this:
<img width="1445" alt="image" src="https://user-images.githubusercontent.com/1480504/214545743-38d928ba-807c-4a12-894b-ca6016a305e7.png">


But that first line shows up in a summary table at the top:

<img width="1466" alt="image" src="https://user-images.githubusercontent.com/1480504/214545644-584363ca-3086-4365-a4f9-aa2fccb7072a.png">